Python Program to Display Fibonacci Sequence Using Recursion.py

```diff
@@ -1,15 +1,16 @@
 def recur_fibo(n):
-   if n <= 1:
-       return n
-   else:
-       return(recur_fibo(n-1) + recur_fibo(n-2))
+    if n <= 1:
+        return n
+    else:
```

Comment:
`else` can be ignored in this case since we have a `return-statement` if its `True` then it'll exit the function, otherwise it'll 100% read the next statement. So another way to make it easier as follows:
```python
if n <=1:
    return n
return recur_fibo(n - 1) + recur_fibo(n - 2)
```